words/objects.txt

```diff
@@ -2334,15 +2973,19 @@ watch
 watcher
 watchmaker
 water
+watercress
 waterfall
+waterliliy
```

Comment:
spelling, I also think water lilies are 2 words?

Reply:
meh, its comprehensible as a single word, so (spelled correctly) I'm ok with it